scripts/prepare-release-artifact.js

```diff
@@ -0,0 +1,123 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+const path = require('path');
+const { Octokit } = require('octokit');
+
+const { version } = require('../package.json');
+
+const WORKFLOW_TIMEOUT_MS = 15 * 60 * 1000;
+
+async function sleep(ms) {
```

Comment:
This doesn't need to be an async function just return a promise, don't await it.